plutus-core/plutus-core/src/PlutusCore/Default/Universe.hs

```diff
@@ -175,6 +175,9 @@ instance DefaultUni `Contains` []            where knownUni = DefaultUniProtoLis
 instance DefaultUni `Contains` (,)           where knownUni = DefaultUniProtoPair
 instance DefaultUni `Contains` Data          where knownUni = DefaultUniData
 
+-- If this tells you a 'Contains' instance is missing, add it right above, following the pattern.
+instance TestTypesFromTheUniverseAreAllContained DefaultUni
```

Comment:
Commenting out the

```haskell
instance DefaultUni `Contains` (,)           where knownUni = DefaultUniProtoPair
```

line makes `TestTypesFromTheUniverseAreAllContained` throw

```
Could not deduce (Contains DefaultUni (,))
```

which is pretty cool if you think about it: `Everywhere` is not poly-kinded and yet we get an error precisely about `(,)` rather than `(a, b)`. This is because the default universe has this instance:

```
instance (DefaultUni `Contains` f, DefaultUni `Contains` a) => DefaultUni `Contains` f a
```

and so GHC does quite a bit of thinking before it presents us with the perfectly precise error message.